docs/sql/data_types/user.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: docu
+title: User Types
```

Comment:
Enum Types? I propose we create a separate page for the other `CREATE TYPE` statement later.